### PR TITLE
feat: add meeting notes for dec 2nd meeting

### PR DESCRIPTION
**Pull request by jithine**
## Context

Add Dec 2nd meeting notes

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: meetings/2019-12/dec-02.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Meeting Notes - 2019-11-18
```
Comment (adong) [inline meetings/2019-12/dec-02.md]: deja vu

File: meetings/2019-12/dec-02.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+## Attendees
+
+- [Jithin Emmanuel](https://github.com/jithine)
+- [Keisuke Kumada](https://github.com/kumada626)
+- [Shota Yoshika](https://github.com/s-yoshika)
+- Yahoo Japan Team
```
Comment (kumada626) [inline meetings/2019-12/dec-02.md]: According to our company guidelines, the company name is written as `Yahoo! JAPAN`. So please rewrite as below.
```suggestion
- Yahoo! JAPAN Team
```

File: meetings/2019-12/dec-02.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+## Agenda
+
+From VMG:
+
+- [Disk Optimzation](https://github.com/screwdriver-cd/screwdriver/issues/1830)
+- [Parallel Fork](https://github.com/screwdriver-cd/screwdriver/issues/1710)
+- [Build timeout not reliable](https://github.com/screwdriver-cd/screwdriver/issues/1710)
+- [Updating dependencies](https://github.com/screwdriver-cd/screwdriver/issues/1722)
+
+From YJapan:
```
Comment (kumada626) [inline meetings/2019-12/dec-02.md]:
```suggestion
From Yahoo! JAPAN:
```

File: meetings/2019-12/dec-02.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+- [Duplicate PR comments](https://github.com/screwdriver-cd/screwdriver/issues/1858)
+- [DB Size is growing](https://github.com/screwdriver-cd/screwdriver/issues/1865)
+
+## Discussion Notes
+
+For [Template/Command list screen shows wrong VERSION](https://github.com/screwdriver-cd/screwdriver/issues/1723) current solution introduces [a bug](https://github.com/screwdriver-cd/screwdriver/issues/1857) with trusted templates. VMG has pinned `datastore-sequelize` to an older version, until this is fixed.
+
+For [Disk Optimzation](https://github.com/screwdriver-cd/screwdriver/issues/1830), discussed workflow, `executor-k8s` to be worked up on after `exececutor-k8s-vm` flow is addressed.
+
+For [Performance issue of webhook](https://github.com/screwdriver-cd/screwdriver/issues/1468), [prs](https://github.com/screwdriver-cd/screwdriver/pull/1850) have been merged, YJapan team will get performance numbers next time.
```
Comment (kumada626) [inline meetings/2019-12/dec-02.md]:
```suggestion
For [Performance issue of webhook](https://github.com/screwdriver-cd/screwdriver/issues/1468), [prs](https://github.com/screwdriver-cd/screwdriver/pull/1850) have been merged, Yahoo! JAPAN team will get performance numbers next time.
```

**Review by kumada626 — COMMENTED** [review]
Please add below topic.
```
For [SD Local](https://github.com/screwdriver-cd/screwdriver/issues/1853), Yahoo! JAPAN team will start with local mode implementation of log-service and launcher. 
```

**Review by kumada626 — APPROVED** [review]
LGTM

**Pull request merged**